docs/CHANGELOG.md

```diff
@@ -1,17 +1,43 @@
 # Change Log
 
+## [0.4](https://github.com/formatc1702/WireViz/tree/v0.4) (2024-XX-XX)
+
+### Backward-incompatible changes
+- New syntax for autogenerated components ([#184](https://github.com/wireviz/WireViz/issues/184), [#186](https://github.com/wireviz/WireViz/pull/186))
+  - Components that are not referenced in any connection set will not be rendered. Instead, a warning will be output in the console. ([#328](https://github.com/wireviz/WireViz/issues/328), [#332](https://github.com/wireviz/WireViz/pull/332))
+- New command line interface ([#244](https://github.com/wireviz/WireViz/pull/244)). Run `wireviz --help` for details 
```

Comment:
You might want to mention both `-h` and `--help` alternatives (or only mention `-h`), but it might be equally good to leave it unchanged (maybe the long option is easier to remember for the inexperienced user).

Reply:
Let's leave it as is here.